### PR TITLE
Add some pointers to runtime installation

### PR DESCRIPTION
**Pull request by generalmimon**
Closes https://github.com/kaitai-io/kaitai_struct/issues/571

https://github.com/kaitai-io/kaitai_struct/issues/699 is related.

### CODE REVIEW

File: _build/usage_python.html.erb [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<p>The Python runtime library can be installed
+<a href="https://pypi.org/project/kaitaistruct/" target="_blank" rel="noopener">from PyPI</a>:</p>
+
+<pre><code>pip install kaitaistruct</code></pre>
```
Comment (dgelessus) [inline _build/usage_python.html.erb:4]: https://snarky.ca/why-you-should-use-python-m-pip/

```suggestion
<pre><code>python3 -m pip install kaitaistruct</code></pre>
```

**Review by dgelessus — COMMENTED** [review]
LGTM - just one minor comment about the Python `pip` call.

**Review by dgelessus — APPROVED** [review]
👍

I'm mainly a Python and Java person, so I can't say much about the instructions for the other languages, but I didn't see anything obviously wrong.

**Review by jchv — APPROVED** [review]
LGTM, and a nice improvement to have, too.

**Comment by generalmimon** [comment]
@dgelessus @jchv Thanks for your reviews! Merging in.

**Pull request merged**